model/src/main/kotlin/OrtResult.kt

```diff
@@ -54,4 +54,10 @@ data class OrtResult(
          */
         @JsonInclude(JsonInclude.Include.NON_EMPTY)
         val data: CustomData = emptyMap()
-)
+) {
+    /**
+     * Conveniently return all detected licenses for the given package [id].
```

Comment:
"Conveniently" seems superfluous.

Reply:
I wanted to emphasize that this function is mainly for scripting convenience, not so much so be called from Main Kotlin code. But I'll remove it anyway.